app/classifier/classification-summary.jsx

```diff
@@ -73,6 +73,15 @@ class ClassificationSummary extends React.Component {
       );
     }
 
+    if (this.props.workflow.configuration &&
+        this.props.workflow.configuration.custom_summary) {
```

Comment:
`custom_summary` is an array earlier in this component, but is a string here.

Reply:
Ignore me, I think I'm getting my string methods and array methods mixed up. There's an earlier check for `custom_summary.includes('worldwide_telescope')`.